lxd/device/device_utils_network.go

```diff
@@ -917,3 +917,21 @@ func networkSRIOVSetupContainerVFNIC(hostName string, config map[string]string)
 
 	return nil
 }
+
+// ExpandNetworkVLANList takes in a list of raw VLAN values (string) that includes
+// different VLAN formats ("number" and "start-end") and convert them into a list of
+// expanded VLAN values in integer.
+func ExpandNetworkVLANList(rawVLANValues []string) ([]int, error) {
```

Comment:
Just a nit, but due to these functions share the same package namespace with other non-network related functions we tend to prefix them with "network" so they come together in an alphabetical function list.

Also, as far as I can see this function isn't called from outside of the `device` package so there's no need to export it.

So please can you call it `networkVLANListExpand()`.